pkg/cri/server/sandbox_run_linux_test.go

```diff
@@ -394,7 +394,8 @@ nameserver server.google.com
 options timeout:1
 `,
 		},
-		"should return error if dns search exceeds limit(6)": {
+		"expanded dns config should return correct content": {
```

Comment:
nit..`on glibc 2.26 and above` For example, in case someone tests it on ubuntu 16.04LTS